hs_core/search_indexes.py

```diff
@@ -80,7 +80,7 @@ class BaseResourceIndex(indexes.SearchIndex, indexes.Indexable):
     # TODO: We might need more information than a bool in the future
     replaced = indexes.BooleanField()
     created = indexes.DateTimeField(model_attr='created')
-    modified = indexes.DateTimeField(model_attr='updated')
+    modified = indexes.DateTimeField(model_attr='last_updated')
     organization = indexes.MultiValueField(faceted=True)
```

Comment:
I still have to do this in communities frontend but does this fix for all places we show last modified in resource listings, such as Discovery, Group Resource listing, My Resources and Resource Landing?

Reply:
This is for the search indexes, below in this file is for Discovery.  I just pushed a change for Group Resource in this branch.  My Resources and Resource Landing were taken care of initially when we discovered the issue.